.github/workflows/rust.yml

```diff
@@ -90,6 +90,47 @@ jobs:
       - name: Run tests
         run: cargo test --locked --workspace --lib --bins --test '*' --exclude fig_desktop-fuzz
 
+  cargo-clippy-windows-chat-cli:
```

Comment:
Split it out this way instead of adding it to the OS Matrix since we only care about the `chat_cli` crate.